projects/plugins/jetpack/extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -61,9 +61,9 @@ public static function render( $attr, $content ) {
 
 		$is_squareish_layout = self::is_squareish_layout( $attr );
 
-		// Jetpack_Plan::get does not exist on WordPress.com.
-		if ( method_exists( 'Jetpack_Plan', 'get' ) ) {
-			$jetpack_plan = Jetpack_Plan::get();
+		// Current_Plan::get does not exist on WordPress.com.
```

Comment:
The comment won't make much sense anymore once this is merged and the package gets loaded on WordPress.com with the rest of the Jetpack plugin.